ideascube/search/migrations/0002_drop_old_index_table.py

```diff
@@ -14,5 +14,5 @@ class Migration(migrations.Migration):
     dependencies = [('search', '0001_initial')]
 
     operations = [
-        migrations.RunPython(migrate),
+        migrations.RunPython(migrate, hints={'using': 'default'}),
```

Comment:
This all looks good as far as the code is concerned.

Since there's nowhere to comment on the commit message I'll put it here:

> django doesn't know on witch

I think you meant "django doesn't know any witch". Or was it « which » ? 😜